NDK build failure

The NDK build bot is failing (all other builds still succeed). The main error is:

```
CMake Error at /home/travis/android-cmake/android.toolchain.cmake:800 (message):
  Specified Android native API level 'android-12' is not supported by your
  NDK/toolchain.
Call Stack (most recent call first):
  /usr/local/cmake-3.9.2/share/cmake-3.9/Modules/CMakeDetermineSystem.cmake:94 (include)
  CMakeLists.txt:38 (project)
```

This is immediately preceded by:

```
CMake Warning at /home/travis/android-cmake/android.toolchain.cmake:323 (message):
  Could not determine machine name for compiler from
  /home/travis/android-ndk/toolchains/mipsel-linux-android-4.9/prebuilt/linux-x86_64
Call Stack (most recent call first):
  /home/travis/android-cmake/android.toolchain.cmake:550 (__DETECT_TOOLCHAIN_MACHINE_NAME)
  /home/travis/android-cmake/android.toolchain.cmake:607 (__GLOB_NDK_TOOLCHAINS)
  /usr/local/cmake-3.9.2/share/cmake-3.9/Modules/CMakeDetermineSystem.cmake:94 (include)
  CMakeLists.txt:38 (project)
```

Which is slightly different than previous successful builds, which instead reported:

```
CMake Deprecation Warning at /usr/local/cmake-3.9.2/share/cmake-3.9/Modules/CMakeForceCompiler.cmake:83 (message):
  The CMAKE_FORCE_CXX_COMPILER macro is deprecated.  Instead just set
  CMAKE_CXX_COMPILER and allow CMake to identify the compiler.
Call Stack (most recent call first):
  /home/travis/android-cmake/android.toolchain.cmake:1140 (CMAKE_FORCE_CXX_COMPILER)
  /usr/local/cmake-3.9.2/share/cmake-3.9/Modules/CMakeDetermineSystem.cmake:94 (include)
  CMakeLists.txt:38 (project)
```

Both good and bad cases are using cmake 3.9.2.

--- a/.travis.yml
+++ b/.travis.yml
@@ -50,8 +50,16 @@ install:
     fi
   # Download Android NDK and Android CMake toolchain file.
   - if [[ "$BUILD_NDK" == "ON" ]]; then
-      git clone --depth=1 https://github.com/urho3d/android-ndk.git $HOME/android-ndk;
+      # Unfortunately the android.toolchain.cmake file does not understand how to
+      # get the API level from Android NDK r17b.  So fall back on r13b.
+      # Get only one commit at tag r13b.
       export ANDROID_NDK=$HOME/android-ndk;
+      git init $ANDROID_NDK;
+      pushd $ANDROID_NDK;
+        git remote add urho3d https://github.com/urho3d/android-ndk.git;
+        git fetch --depth=1 urho3d r13b;
+        git checkout FETCH_HEAD;
+      popd;
       git clone --depth=1 https://github.com/taka-no-me/android-cmake.git $HOME/android-cmake;
       export TOOLCHAIN_PATH=$HOME/android-cmake/android.toolchain.cmake;
     fi

